edenia/antelope-tools
# Chore/change order of networks 1222

### Change order of networks

### What does this PR do?

- Resolve #1222 
- Avoid to check the account name in the ultra testnet faucet

### Steps to test

1. Run the project locally
2. Check the order of the list in the network selector
3. Go to homepage
4. Check the order of the cards

### Screenshots

## Network selector

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/f6471a26-1e45-4eb4-8dd4-ee905164334c)


## Homepage

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/ad4c8b9c-2ee3-4d5d-bf94-82b584acfe18)
